agent/agent-gc-monitor/gc-monitor-core/src/main/java/com/microsoft/gcmonitor/JMXGCMonitorFactory.java

```diff
@@ -0,0 +1,20 @@
+package com.microsoft.gcmonitor;
+
+import javax.management.MBeanServerConnection;
+import java.lang.management.ManagementFactory;
+import java.util.concurrent.ExecutorService;
+
+public class JMXGCMonitorFactory implements GcMonitorFactory {
```

Comment:
I've ended up liking this style:
```suggestion
public class JmxGcMonitorFactory implements GcMonitorFactory {
```